static-assets/components/cstudio-components/dropbox.js

```diff
@@ -593,7 +593,11 @@ if (typeof HTMLElement != "undefined" && !HTMLElement.prototype.insertAdjacentEl
                 me.manager.setCompleted(file.id);
                 me.showUploadProgress( elemProgress, 100, fileUI );
 
+              if (document.querySelector('.bulk-upload .buttons-container .close') &&
+                (Object.keys(me.manager.files).length === Object.keys(me.manager.complete).length)
+              ) {
                 document.querySelector(".bulk-upload .buttons-container .close").classList.remove("disabled");
+              }
```

Comment:
Adding/removing a disabled class is not enough to enable/disable a button. Set enabled/disabled accordingly.

Why do you need to check for the existence of the button?

Reply:
because, the modal can be closed with esc btn